.travis.yml

```diff
@@ -40,4 +40,11 @@ deploy:
     script: if [ "$GH_TOKEN" != "false" ]; then npx auto shipit; fi;
     skip-cleanup: true
     on:
-      all_branches: true
\ No newline at end of file
+      all_branches: true
+  - provider: pages
+    local-dir: "./storybook-static"
```

Comment:
Are you sure storybook is generated in the root folder and not in the doc folder? I'm not sure.

Reply:
I'm not really sure what "./storybook-static" means..I didn't find storybook-static in react-multi-select
In react tree the storybook is created inside doc folder, how can we know where is the storybook generated?